manifests/ssltools/nssdb.pp

```diff
@@ -6,7 +6,15 @@
   Exec { logoutput => 'on_failure' }
 
   $nss_db_password_file   = "${nss_db_dir}/nss_db_password-file"
-  $nssdb_files            = ["${nss_db_dir}/cert8.db", "${nss_db_dir}/key3.db", "${nss_db_dir}/secmod.db"]
+
+  case $operatingsystemrelease {
```

Comment:
You could also use `$operatingsystemmajrelease` which is only the major version. Nowadays we should also use `$facts['operatingsystemmajrelease']`.

Reply:
Updated